code/modules/mob/living/simple_animal/abnormality/zayin/bald.dm

```diff
@@ -17,6 +18,17 @@
 	work_damage_amount = 4
 	work_damage_type = WHITE_DAMAGE
 
+	melee_damage_lower = -1
+	melee_damage_upper = -1
+	melee_damage_type = WHITE_DAMAGE
+	attack_verb_continuous = "balds"
+	attack_verb_simple = "bald"
+
+	ranged = TRUE
+	ranged_message = "balds"
+	projectiletype = /obj/projectile/beam/yang/bald
```

Comment:
Why is it a subtype of yang's projectile..?

Reply:
Minimized code copy-paste. Most of it is literally just coloring/icons.